### PR TITLE
added runs for compiler

### PR DESCRIPTION
**Pull request by Aniket-Engg**
Related to #320

### CODE REVIEW

File: libs/remix-solidity/src/compiler/types.ts [hunk 1]
```diff
@@ -143,6 +143,7 @@ export interface Source {
 
 export interface CompilerInputOptions {
     optimize: boolean | number,
+    runs: number,
```
Comment (GrandSchtroumpf) [inline libs/remix-solidity/src/compiler/types.ts:146]: Can it be any number or a subset of number ?
Let's take an example, if you want to display the status of a request you might want to use number literal :
```typescript
status: 200 | 400 | 500 | ...;
```
I'm not sure if it's the same in your case though.

**Review by GrandSchtroumpf — APPROVED** [review]

File: libs/remix-solidity/src/compiler/types.ts (hunk 1 above)
Comment (Aniket-Engg) [reply libs/remix-solidity/src/compiler/types.ts:146]: Usually It is 200. Further user can define it from 1 to bigger value

Comment (GrandSchtroumpf) [reply libs/remix-solidity/src/compiler/types.ts:146]: ok, then keep it at number

Comment (LianaHus) [reply libs/remix-solidity/src/compiler/types.ts:146]: do you know the range if any? could it be 10000000? @Aniket-Engg

**Pull request merged**